server/src/main/java/org/jetbrains/bsp/bazel/server/sync/BspProjectMapper.java

```diff
@@ -107,12 +113,18 @@ private BuildTarget toBuildTarget(Module module) {
   }
 
   private BuildTargetCapabilities inferCapabilities(Module module) {
+    checkManualTag(module);
     var canCompile = !module.tags().contains(Tag.NO_BUILD);
     var canTest = module.tags().contains(Tag.TEST);
     var canRun = module.tags().contains(Tag.APPLICATION);
     return new BuildTargetCapabilities(canCompile, canTest, canRun);
   }
 
+  private WorkspaceContext checkManualTag(Module module) {
```

Comment:
same here - the method does nothing. it should check the condition (described below) and be used in the calculation of `canCompile` - currently it isnt used there